### ISSUE TITLE
[FSR] Update the dropdown date component with the experimental design system's input date component

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#40714]
## Issue Description
A new text date input component is being added to the experimental design system. We should update the dropdown date components in the FSR to the new component so we can test it as part of the combined VHA/VBA FSR project. 

Information about the component can be found in ticket [#481](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/481).

---
## Tasks
- [ ] Update the dropdown date components in the FSR with the text input date component

## Acceptance Criteria
- [ ] The FSR's UXPin has been updated and linked in this ticket

**Comment by rileyorr** [comment]
Update on this work: I reached out in the ticket about the component to see if it's currently available in the design system to reference in our prototype.

**Comment by rileyorr** [comment]
The design system team notified me that the component is currently unavailable. However their team has sprint planning this Thursday (May 12th) and will add the component's development as a stretch goal. Current estimate is 3 weeks, which aligns with our timeline since we estimate at least a month before we can conduct the usability study for the combined FSR. 

Moving this ticket to blocked for now.

**Comment by denisecoveyduc** [comment]
Nov 2,22 - moving to icebox until VSP Platform team has completed the component

**Comment by denisecoveyduc** [comment]
1/18/23- FE team confirmed that they are using this component as part of FSR enhancement designs.
